bouncer/src/repo/manipulator/modelconvertor/import/odaHelper/geometry_dumper_rvt.h

```diff
@@ -0,0 +1,71 @@
+/**
+*  Copyright (C) 2018 3D Repo Ltd
+*
```

Comment:
Once the materials stuff are all done and triangulation sorted, you will probably find there is a lot of similarity with the dgn implementation and we probably should have a parent class that inherits OdGiGeometrySimplifier keeping the duplicated code.

Leave it for now, but things to bear in mind for final clean up.